src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  background-color: $white-color;
+  box-sizing: border-box;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  transition: transform 300ms;
+
+  @include hover(transform, scale(1.2));
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    color: $main-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 16px 0;
+
+    @include hover(color, #34568b);
+  }
+
+  &__code {
+    color: $secondary-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-top: 4px;
+    margin-left: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    margin: 8px 16px 0 18px;
```

Comment:
The values '8px', '16px', '18px' seem to be arbitrary. Consider using variables or calculations to make it clear where these numbers come from.